client/src/components/ScoreForm.js

```diff
@@ -1,17 +1,31 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { Table, Form, Button, Checkbox, Icon, Popup } from 'semantic-ui-react';
+import { bindActionCreators } from 'redux';
+import { ScoreFormActions } from '../actions';
```

Comment:
Why are these in braces vs. the way you imported react?

Reply:
That's ES6 syntax to just pull that specific object/function rather than the entire file. Makes it more efficient. For example, rather than having the entire semantic-ui-react file, I'm only using specific functions from it like Table or Form.